routes/confirmation/checkAnswersFormat.js

```diff
@@ -0,0 +1,65 @@
+const { answerInfo } = require('./checkAnswers')
+const { hasData } = require('./../../utils')
+const { format, parseISO } = require('date-fns')
+const { routes } = require('./../../config/routes.config')
+
+const formatDataLine = (data, session) => {
+  if (data.length > 1) {
+    const dataValues = []
+    data.map((val) => {
+      dataValues.push(hasData(session, val, true))
+    })
+
+    if (typeof dataValues[1] === 'number') {
+      return dataValues.reduce((a,b) => a + b, 0)
+    } else if (typeof dataValues[1] === 'string') {
+      return dataValues.join(' ')
+    }
+
+  } else {
+    if(hasData(session,data[0],true) === null || hasData(session,data[0],true) === false) {
+      return 'No'
+    } else if (hasData(session,data[0],true) === true) {
+      return 'Yes'
+    } else if (data[0].includes('Birth')) {
+      const initialDate = parseISO(hasData(session,data[0],true))
+      return format(new Date(initialDate), 'd MMMM yyyy')
+    } else {
+      return hasData(session,data[0],true)
```

Comment:
Ah, sorry, I meant to do something like this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch

I think it would be about as much work but cleaner — (but up to you).